src/oscar/templatetags/form_tags.py

```diff
@@ -24,6 +26,6 @@ def __init__(self, field_str):
 
     def render(self, context):
         field = self.field.resolve(context)
-        if hasattr(field, 'field'):
-            field.widget_type = field.field.widget.__class__.__name__
+        if not hasattr(field, 'widget_type') and hasattr(field, 'field'):
```

Comment:
Could we please move this class to `oscar/core/compat.py` with a comment explaining that the tag will be redundant after we drop support for Django 2.2 and lower? That way we will not forget to clean things up when it does become redundant.

Reply:
Done in 663b8eb